textproc/OCRmyPDF/Portfile

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        ocrmypdf OCRmyPDF 16.1.2 v
+github.tarball_from releases
+distname            ocrmypdf-${github.version}
+
+categories          textproc
+maintainers         {@akierig fastmail.de:akierig} openmaintainer
+revision            0
+
+checksums           rmd160  25b94fbd304d1fb8235cbd12c1b8fb2af0cf868a \
+                    sha256  d1f9c62b3a0cb090218e8f719d85d110bad95cb73e83dec05842652d173b0c29 \
+                    size    6720680
+
+description         ${name} adds an OCR text layer to scanned PDF files, \
+                    allowing them to be searched
+long_description    ${description}. ${name} OCRmyPDF also supports plugins \
+                    that enable customization of its processing steps, \
+                    and it is highly tolerant of PDFs containing scanned \
+                    images and “born digital” content that doesn’t require text recognition.
+
+supported_archs     noarch
+platforms           {darwin any}
+license             GPL-2
+python.default_version 312
+
+depends_build-append port:py${python.version}-setuptools \
```

Comment:
the py-setuptools dependency is implied as you're using the default value (i.e., "setuptools") for python.pep517_backend" so it can/should be removed here.